src/components/[guild]/activity/ActivityLogAction/components/MoreActions.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Box,
+  HStack,
+  Icon,
+  ListItem,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Text,
+  UnorderedList,
+  useDisclosure,
+} from "@chakra-ui/react"
+import Button from "components/common/Button"
+import { Modal } from "components/common/Modal"
+import { ArrowSquareOut, DotsThree } from "phosphor-react"
+import { CSSProperties, memo } from "react"
+import { FixedSizeList } from "react-window"
+import { ActivityLogAction } from "../../constants"
+import { ActivityLogActionProvider } from "../ActivityLogActionContext"
+import ActionIcon from "./ActionIcon"
+import ActionLabel from "./ActionLabel"
+
+type Props = {
+  actions: ActivityLogAction[]
+  displayedActionCount: number
+}
+
+const MoreActions = ({ actions, displayedActionCount }: Props): JSX.Element => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const numberOfAdditionalActions = actions.length - displayedActionCount
+
+  const Row = memo(({ index, style }: { index: number; style: CSSProperties }) => {
+    const action = actions[index]
+
+    return (
+      <ListItem style={style}>
+        <ActivityLogActionProvider key={action.id} action={action}>
+          <Box
```

Comment:
I think we should make the whole container horizontally scrollable, not just the lines by themeselves, and that way this solution is sufficient

Reply:
Actually, after experimenting with multiple solution, I felt like making a variable size list is a better solution, so I implemented it in 1c2a33bf062810f2b8e2122bc040d3fcb7c0db7f (note: the list item heights don't update on window resize events, but I thought that's a rare edge case)